include/boost/multiprecision/mpfr.hpp

```diff
@@ -2386,7 +2387,7 @@ class numeric_limits<boost::multiprecision::number<boost::multiprecision::mpfr_f
    BOOST_STATIC_CONSTEXPR int digits   = static_cast<int>((Digits10 * 1000L) / 301L + ((Digits10 * 1000L) % 301 ? 2 : 1));
    BOOST_STATIC_CONSTEXPR int digits10 = Digits10;
    // Is this really correct???
-   BOOST_STATIC_CONSTEXPR int  max_digits10 = Digits10 + 3;
+   BOOST_STATIC_CONSTEXPR int  max_digits10 = boost::multiprecision::detail::calc_max_digits10<digits>::value;
```

Comment:
sorry, in my previous comment I was talking about this `+ 3` in fact, and the `+1` in `max_digits_10` :)

Reply:
BTW, that comment "`// Is this really correct???`" in this place is quite accurate :)

Reply:
LOL, yes I know!

I wish I could find a definitive reference for this somewhere, but it eludes me :(